samples/csharp/getting-started/Ranking_AutoML/Ranking/README.md

```diff
@@ -0,0 +1,206 @@
+## Automated Machine Learning
+
+Automated machine learning (AutoML) automates the end-to-end process of applying machine learning to real-world problems. Given a dataset, AutoML iterates over different data featurizations, machine learning algorithms, hyperparamters, etc. to select the best model.
+
+## Problem
+The ability to perform ranking is a common problem faced by search engines since users expect query results to be ranked/sorted according to their relevance. This problem extends beyond the needs of search engines to include a variety of business scenarios where personalized sorting is key to the user experience. Here are a few specific examples:
+* Travel Agency - Provide a list of hotels with those that are most likely to be purchased/booked by the user positioned highest in the list.
+* Shopping - Display items from a product catalog in an order that aligns with a user's shopping preferences.
+* Recruiting - Retrieve job applications ranked according to the candidates that are most qualified for a new job opening.
+
+Ranking is useful to any scenario where it is important to list items in an order that increases the likelihood of a click, purchase, reservation, etc.
+ 
+In this sample, we show how to apply ranking to search engine results. To perform ranking, there are two algorithms currently available - FastTree Boosting (FastRank) and Light Gradient Boosting Machine (LightGBM). We use the LightGBM's LambdaRank implementation in this sample to automatically build an ML model to predict ranking. 
```

Comment:
```suggestion
In this sample, we show how to apply ranking to search engine results. To perform ranking, there are two algorithms currently available - FastTree Boosting (FastRank) and Light Gradient Boosting Machine (LightGBM). AutoML sweeps over the models and associated hyperparameters to optimize [NDCG@N](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.data.rankingmetrics.normalizeddiscountedcumulativegains?view=ml-dotnet) (normalized discounted cumulative gains) or [DCG@N](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.data.rankingmetrics.discountedcumulativegains?view=ml-dotnet) (discounted cumulative gains).
```

Notation note: I also used `@N` in the text here, as our docs reference `NDCG@N` (in other places, I was using `@k`).